include/aikido/rviz/InteractiveMarkerViewer.hpp

```diff
@@ -34,22 +41,29 @@ class InteractiveMarkerViewer
 
   interactive_markers::InteractiveMarkerServer& marker_server();
 
-  SkeletonMarkerPtr addSkeleton(const dart::dynamics::SkeletonPtr& skeleton);
-
-  FrameMarkerPtr addFrame(
+  /// Visualizes a Skeleton.
+  /// \param skeleton Skeleton to add to the viewer
+  /// \return Skeleton marker added to the viewer.
+  SkeletonMarkerPtr addSkeletonMarker(
```

Comment:
So if the Viewer is initialized without `mWorld`, can one still call this function? If not, can we update the docstring and set a check inside the function.

Reply:
Yes.